library/src/main/scala/scala/scalajs/runtime/LinkingInfo.scala

```diff
@@ -0,0 +1,32 @@
+package scala.scalajs.runtime
+
+import scala.scalajs.js
+
+/** Information about link-time configuration of Scala.js. */
+trait LinkingInfo extends js.Object {
```

Comment:
Is there a reason you are not defining everything we actually expose on this trait?